src/main.cpp

```diff
@@ -10,7 +10,7 @@ int main(int argc, char** argv)
                   "Usage: coolstory_gram_client <host> <port> <text>\n" <<
                   "Example:\n" <<
                   "    coolstory_gram_client echo.websocket.org 80 \"Hello, world!\"\n";
-        return EXIT_FAILURE;
+        return EXIT_SUCCESS;

```

Comment:
Need to fix build scripts, but not the code.
Change run command and add `--help` flag. Of course add `--help` flag processing in the app.

Reply:
We will count the number of arguments instead of parsing them.